content/wiki/projects.toml

```diff
@@ -253,6 +253,15 @@ state = "Beta"
 ################### Active Development ####################
 ###########################################################
 
+[[projects]]
+name = "Polaron World Engine"
+url = "https://polaronworldengine.com/"
+state = "Active Dev."
+source = "https://gitlab.com/veloren/veloren"
+license = "Commercial"
+language = ["C++"]
+rendering = "Ray Marching, Ray Casting"
+
```

Comment:
New projects are supposed to go at the bottom of their respective status section (Active Development in this case).